core/src/main/scala/org/http4s/headers/User-Agent.scala

```diff
@@ -2,38 +2,29 @@ package org.http4s
 package headers
 
 import org.http4s.parser.HttpHeaderParser
-import org.http4s.util.{Renderable, Writer}
+import org.http4s.util.Writer
 
 object `User-Agent` extends HeaderKey.Internal[`User-Agent`] with HeaderKey.Singleton {
+  def apply(id: ProductId): `User-Agent` =
```

Comment:
Shouldn't `Server` have this same constructor?  Or just make a default arg of `Nil`?